src/Casdoor.Client/Abstractions/ICasdoorServiceClient.cs

```diff
@@ -17,7 +17,8 @@ namespace Casdoor.Client;
 public interface ICasdoorServiceClient
 {
     public Task<CasdoorResponse?> SendSmsAsync(string content, params string[] receivers);
+    public Task<CasdoorResponse?> SendSmsAsync(string content, CancellationToken cancellationToken , params string[] receivers);
```

Comment:
We need to try our best to ensure the compilation compatibility of users, and the cancellationToken should always at the last.
We can provide another one like,
```csharp
    public Task<CasdoorResponse?> SendSmsAsync(string content, IEnumerable<string> receivers, CancellationToken cancellationToken);
```

Reply:
done